src/components/IconButton.js

```diff
@@ -0,0 +1,41 @@
+import styled, { css } from 'styled-components';
+import ScreenReaderOnly from './ScreenReaderOnly';
+
+export default function IconButton({ buttonText, children, ...buttonProps }) {
+  return (
+    <ButtonStyled {...buttonProps}>
+      {children}
+      <ScreenReaderOnly>{buttonText}</ScreenReaderOnly>
+    </ButtonStyled>
+  );
```

Comment:
I think you can delete this here since you added it as children in the `Card.js`:
```suggestion
export default function IconButton({ children, ...buttonProps }) {
  return (
    <ButtonStyled {...buttonProps}>
      {children}
    </ButtonStyled>
  );
```